crypto/x509/x509_obj.c

```diff
@@ -117,7 +117,7 @@ char *X509_NAME_oneline(X509_NAME *a, char *buf, int len)
             type == V_ASN1_PRINTABLESTRING ||
             type == V_ASN1_TELETEXSTRING ||
             type == V_ASN1_VISIBLESTRING || type == V_ASN1_IA5STRING) {
-            ascii2ebcdic(ebcdic_buf, q, (num > sizeof ebcdic_buf)
+            ascii2ebcdic(ebcdic_buf, q, ((sizeof ebcdic_buf) < num)
```

Comment:
@kroeckx I intentionally added those braces to clarify the intent - that it's not

     (sizeof ebcdic_buf) < ( num ? sizeof ebcdic_buf : num).

Do they really hurt?